skada/tests/test_self_labeling.py

```diff
@@ -66,6 +66,7 @@ def test_dasvm_estimator_predict_proba():
         shift="covariate_shift",
         noise=None,
         label="binary",
+        random_state=0,
```

Comment:
```suggestion
        random_state=None,
```
?

Reply:
sorry I did not see this was   atest

Reply:
all good 👍 it would be likely better to use 42, which is common for other tests cases